packages/flutter/lib/src/animation/tween_sequence.dart

```diff
@@ -1,3 +1,7 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Technically, I think the year to use is the one in which this file was originally authored, not the day the copyright notice was added. You can find this out in the git log for the file.

Reply:
Changed to 2018 according to the earliest commit in git log.